src/test/java/IMDbTop250Scraper.java

```diff
@@ -0,0 +1,92 @@
+import com.codeborne.selenide.*;
+ import java.util.ArrayList;
+import java.util.List;
+import static com.codeborne.selenide.Selenide.*;
+
+public class IMDbTop250Scraper {
+
+    public static void main(String[] args) {
+        configureSelenide();
+        openImdbPage();
+        openNavigationMenu();
+        navigateToTop250();
+        List<String> movieData = scrapeMovies(20);
+        printTopMovies(movieData);
+        double averageRating = calculateAverageRating(movieData);
+        printAverageRating(averageRating);
+    }
+
+    private static void configureSelenide() {
+        Configuration.timeout = 10000;
+    }
+    private static void openImdbPage() {
+        open("https://www.imdb.com");
+    }
+    private static void openNavigationMenu() {
+        $("#imdbHeader-navDrawerOpen").should(Condition.exist).click();
```

Comment:
Can be just $("#imdbHeader-navDrawerOpen").should(exist).click();
but i would use should... as it is like a waiting method, waits until something appears... more bulletproof I'd say..
$("#imdbHeader-navDrawerOpen").shouldBe(visible).click();

+ maybe just private static void openMenu()